### PR TITLE
updated app to run with cosmosdb

### PR DESCRIPTION
**Pull request by MHolmes219**
Please make sure you are raising this PR against your own repository and not the original. If it says https://github.com/CorndelWithSoftwire in the search bar right now, you are in the wrong place!

Please also don't forget to submit the PR's url to Aptem afterwards (by clicking on the corresponding exercise component, pasting the url in the textbox that appears and then pressing the finish button).

### CODE REVIEW

File: tests/test_app.py [hunk 1]
```diff
@@ -26,20 +22,9 @@ def json(self):
...
-        }]
-    return StubResponse(fake_response_data)
-
-
-def test_index_page(monkeypatch, client):
-    monkeypatch.setattr(requests, 'get', get_lists_stub)
-    response = client.get('/')
-
-    assert response.status_code == 200
-    assert 'Test card' in response.data.decode()
```
Comment (BenRamchandani) [inline tests/test_app.py]: You've deleted your integration test!

You will need to change it a bit:

* We don't need any of the stub or monkey patch stuff,
* Instead we will first add a card to our fake mongo database, by connecting to it and inserting one
* Then we can test that that card appears in the response in the same way this test did

File: todo_app/data/trello_items.py [hunk 2]
```diff
@@ -3,25 +3,34 @@
...
+
+def __get_database():
+    database = __get_endpoint()
+    database = database[app.config['DATABASE']]
+    return database
+
+def __get_list_collection():
+    database = __get_database()
+    lists = database.lists
+    return lists
```
Comment (BenRamchandani) [inline todo_app/data/trello_items.py]: This, `get_list` and `get_lists` are not used

File: todo_app/data/trello_items.py [hunk 3]
```diff
@@ -177,28 +182,26 @@ def update_card_list(cardId, list):
...
     """
-    params = __build_params({ 'idList': list['id'] })
-    url = __build_url('/cards/%s' % cardId)
 
-    response = requests.put(url, params = params)
-    card = response.json()
+    cards = __get_card_collection()
 
-    return card
+    response = cards.update_one({'_id': ObjectId(cardId)}, {'$set': {'list': list}})
```
Comment (BenRamchandani) [inline todo_app/data/trello_items.py]: It would be nice to update `dateLastActivity` too

File: todo_app/flask_config.py [hunk 4]
```diff
@@ -11,3 +11,6 @@ def __init__(self):
         self.API_KEY = os.environ.get('API_KEY')
         self.API_TOKEN = os.environ.get('API_TOKEN')
         self.BOARD_ID = os.environ.get('BOARD_ID')
```
Comment (BenRamchandani) [inline todo_app/flask_config.py]: You can delete all the trello stuff

File: .env.test [hunk 5]
```diff
@@ -7,4 +7,7 @@ FLASK_SKIP_DOTENV=True
 SECRET_KEY=secretkey
 API_KEY=12345abcde
 API_TOKEN=1234567890
-BOARD_ID=61beadyeyes40
\ No newline at end of file
+BOARD_ID=61beadyeyes40
```
Comment (BenRamchandani) [inline .env.test]: You can delete the trello stuff (leave SECRET_KEY, you'll need it in the next module)

Can you update .env.template too?

**Review by BenRamchandani — DISMISSED** [review]
Your app looks good and all works, the only thing missing is your integration test. I've left a little bit of guidance in a comment below, but feel free to contact us if it doesn't make sense.

I've left some other optional comments, mostly around tidying up and removing references to Trello.

**Review by JackMeadDev — APPROVED** [review]
Thanks for making that change, I can see the integration test is happily passing in your pipeline now! Happy for you to merge this when you're ready

**Pull request merged**